cardano-api/src/Cardano/Api/CliMode.hs

```diff
@@ -27,9 +27,9 @@ import           Cardano.Api.Typed
 import           Ouroboros.Consensus.Node.Run (RunNode)
 
 
-data Protocol = ByronProtocol !EpochSlots
-              | ShelleyProtocol
-              | CardanoProtocol !EpochSlots
+data CliMode = CliByronMode !EpochSlots
+             | CliShelleyMode
+             | CliCardanoMode !EpochSlots
```

Comment:
I don't really agree with this name change. `Protocol` still seems appropriate imo.

Reply:
I agree. Lets not change the names in this PR.

I also have a good solution that avoids having to come up with new names, and will not clash with the existing `NodeConsensusMode` type. But we can do that later.